tests/test_commit_message.py

```diff
@@ -85,15 +85,21 @@ def test_commit_message_blocking_reviewers_requested(reviewer_text):
         "r?#group1",
         "r?#group1, #group2",
         "r?reviewer_one,#group1",
-        "r?#group1 r?reviewer_two",
+        "r?#group1 r?reviewer.two",
         "r?#group1! r?group2",
         "r?#group1 r?group2!",
         "r?#group1! r?group2!",
-        "r?#group1, reviewer_two!",
+        "r?#group1, reviewer.two!",
+        "r=.a",
+        "r=..a",
+        "r=a...a",
+        "r=a.b",
+        "r=a.b.c",
+        "r=aa,.a,..a,a...a,a.b,a.b.c",
     ],
 )
 def test_group_reviewers_replaced(reviewer_text):
```

Comment:
Nit: This test could maybe use a rename now that it's not just focused on groups.

Reply:
Done. Also consolidated another test into this one.